services/app-web/src/pages/Errors/ErrorBoundaryRoot.tsx

```diff
@@ -18,7 +18,6 @@ function ErrorBoundaryRoot({
 }): React.ReactElement {
     useEffect(() => {
         const logError = async () => {
-            await recordSpan('React: ErrorBoundary')
             recordJSException(
```

Comment:
In `App.tsx` this component is outside the `TracingInitializer` context. Should we use `recordJSExceptionWithContext` (assuming the name does what it says) here instead?

Reply:
Yeah, exactly -- we have the `TracingInitializer` that does the `setGlobalTracingContext`. The `recordJSExceptionWithContext` function is something we've had for a while, to which I just added additional span recordings. I wanted us to not have to rip out all of the existing `record` function calls everywhere in `app-web` for now, which we can clean up in a follow up ticket.